src/components/profile-content/profile-content.js

```diff
@@ -226,8 +225,8 @@ const ProfileContent = (props) => {
             excerpt: r.excerpt,
             htmlAst: r.htmlAst,
             type: success.likesDislikedRules
-              .filter((v) => v.ruleGuid == r.frontmatter.guid)
-              .map((r) => r.type),
+              .filter((v) => v.RuleGuid == r.frontmatter.guid)
```

Comment:
Interesting.... are all the json coming back with uppercase. If so.... maybe we should fix this on the backend as to make sure everything stays the same. Or are there only a few references to these properties? 

1. Please investigate if we should continue converting these to uppercase or if we should fix the az functoins to return camel case